# This document was autogenerated by the script camera_robot_calibration.py
# EDITING THIS FILE BY HAND IS NOT RECOMMENDED
# NOTE 1: This is the transformation between the camera coordinates and the robot reference system.
# NOTE 2: The rotation is a quaternion [w,x,y,z]
camera_robot:
  angle_axis: [-2.9626288886630676, -0.6629986744666475, 0.7486097860290238, -0.004018198233990024]
  child: camera_optical_frame
  parent: torso_base_link
  rotation: [-0.6603461274794603, 0.7456147232800557, -0.004002122093825714, -0.08936251658068028]
  samples: 15
  translation: [0.5582442738305349, 0.07349373222024524, 1.5982606898601994]
rmse: {rotation: 0.08010828496732389, translation: 0.06459073933774262}
